File: 프로그래머스/0/181894. 2의 영역/2의 영역.java
class Solution {
    public int[] solution(int[] arr) {
        int start = -1;
        int end = -1;
        for(int i = 0; i < arr.length; i++){
            if(start == -1 && arr[i] == 2) {start = i; end = i;}
            if(start != -1 && arr[i] == 2) end = i;
        }
        int[] ret = new int[end - start + 1];
        if(start == -1) return new int[]{-1};
        for(int i = start; i <= end; i++){
            ret[i-start] = arr[i];
        }
        return ret;
    }
}

File: 프로그래머스/0/181894. 2의 영역/2의 영역.py
def solution(arr):
    start = -1
    end = -1
    for i, num in enumerate(arr):
        if num == 2 and start == -1:
            start, end  = i, i 
        elif num == 2 and start != -1:
            end = i
    ret = arr[start: end + 1]
    return ret if ret else [-1]
            